scripts/datacontainer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.8
+
+RUN apk update && \
+    apk add git
+
+RUN mkdir -p /usr/data/specrepo/
+
+WORKDIR /usr/data/
+RUN git clone https://github.com/sanjaiganesh/azure-rest-api-specs.git ./specrepo
```

Comment:
Change the source git repo or maybe pass it from pipeline yml variable. See https://docs.docker.com/engine/reference/builder/#arg

Reply:
Since you have the git repository in the image, it doesn't matter which branch you have actually. You can build one image for every branch.

Reply:
yup. thanks. updated to just copy .git folder only. switching branches in run.sh controlled via env variable